herbert_core/classes/MPIFramework/mess_cache.m

```diff
@@ -25,27 +22,42 @@
     end
     
     
-    methods(Static)
-        function obj = instance(varargin)
-            persistent cache;
-            if nargin>0
-                if ischar(varargin{1}) % clear cache holder
-                    clear cache;
-                    cache = [];
-                    return;
-                else
-                    argi = varargin;
-                end
-            else
-                argi = {};
+    %     methods(Static)
+    %         function obj = instance(varargin)
```

Comment:
disabled code

Reply:
can be removed, was not sure if it needs disabling or not. Now shure it can be removed.